modules/vrtcalBidAdapter.js

```diff
@@ -21,10 +22,32 @@ export const spec = {
         }
       }
 
+      let gdprApplies = 0;
+      let gdprConsent = '';
+      let ccpa = '';
+      let coppa = 0;
+      let tmax = 0;
+
+      if (bid && bid.gdprConsent) {
+        gdprApplies = bid.gdprConsent.gdprApplies ? 1 : 0;
+        gdprConsent = bid.gdprConsent.consentString;
+      }
+
+      if (bid && bid.uspConsent) {
+        ccpa = bid.uspConsent;
+      }
+
+      if (config.getConfig('coppa') === true) {
+        coppa = 1;
+      }
+
+      tmax = config.getConfig('bidderTimeout');
+
       const params = {
         prebidJS: 1,
         prebidAdUnitCode: bid.adUnitCode,
         id: bid.bidId,
+        tmax: tmax,
```

Comment:
this should probably be `bidderRequest.timeout`. it's equivalent except when the publisher specifies the timeout manually with [`pbjs.requestBids({timeout: ...})`](https://docs.prebid.org/dev-docs/publisher-api-reference/requestBids.html), in which case this would miss it. 

I don't know how much it matters since the correct timeout is enforced by core anyway - it depends on what the backend does with it.